### PR TITLE
Branding compliance updates

### LINKED ISSUE #35: branding compliance
I need to make sure all references to F5, F5 Networks, and F5 products have the appropriate [trademarks](https://f5.com/about-us/policies/trademarks).

### PR DESCRIPTION
**Pull request by jputrino**
@pjbreaux 

Fixes #35 
#### What's this change do?
- made sure all non-code references to F5, F5 Networks, and F5 products use the appropriate trademarks.
- light edits to README, index.rst, CONTRIBUTING
- added venv/\* to .gitignore
#### Where should the reviewer start?

review the changes below
#### Any background context?

I ran flake8 locally and didn't get any errors, so I think everything should be good. We'll see what Travis says.

### CODE REVIEW

**Comment by jputrino** [comment]
I added an encode line (`coding=utf-8`) to the top of the files for all of the modules and mixins.py; this was necessary because the use of the non-ASCII trademark character was breaking pytest.

File: requirements.txt [hunk 1]
```diff
@@ -1 +0,0 @@
-f5-sdk>=0.1.2
```
Comment (pjbreaux) [inline requirements.txt:1]: Why was this removed?

File: f5_heat/resources/test/test_f5_ltm_pool.py [hunk 2]
```diff
@@ -12,6 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+#
```
Comment (pjbreaux) [inline f5_heat/resources/test/test_f5_ltm_pool.py:17]: Is this the standard for our copyright?

File: requirements.txt (hunk 1 above)
Comment (jputrino) [inline requirements.txt:1]: Hmm, I don't remember actually removing that. Adding it back in.

File: f5_heat/resources/test/test_f5_ltm_pool.py (hunk 2 above)
Comment (jputrino) [inline f5_heat/resources/test/test_f5_ltm_pool.py:17]: As far as I know!
